workflow-templates/validate_pr.yml

```diff
@@ -30,26 +30,37 @@ jobs:
       - name: Check for changed source files
         id: changed
         run: |
-          echo "::set-output name=files::$( \
-            git diff --name-only --diff-filter=ACMRT \
+          {
+            echo "files=$(git diff --name-only --diff-filter=ACMRT \
             ${{ github.event.pull_request.base.sha }} \
-            ${{ github.sha }} -- './**/*.xml,!./**/transkribus*.xml,!./**/facsimile*.xml' | xargs)"
+            ${{ github.sha }} \
+            -- '*.xml' '**/*.xml' ':!*/transkribus*.xml' ':!*/facsimile*.xml' | xargs)"
+          } >> "$GITHUB_OUTPUT"
+      # If no xml files were modified ...
+      - name: No source files changed
+        if: ${{ steps.changed.outputs.files == '' }}
+        run: |
+          echo "All good"
+      # ... otherwise
+      - name: List modified documents
+        env:
+          FILES: ${{ steps.changed.outputs.FILES }}
+        if: ${{ steps.changed.outputs.FILES != '' }}  
+        run: |
+          echo "modified: $FILES"
       # Validate changed files against latest schema
       - name: Validate changed source files
+        env:
+          FILES: ${{ steps.changed.outputs.FILES }}
         if: ${{ steps.changed.outputs.files != '' }}
         run: |
           xmllint \
             --noout --xinclude --nowarning \
             --relaxng ../schema/tei-betamesaheft.rng \
-            ${{ steps.changed.outputs.files }} \
+            $FILES \
             2>&1 | \
             { grep -vE 'RNG|validates' || :; }
-      # If no xml files were modified
-      - name: No source files changed
-        if: ${{ steps.changed.outputs.files == '' }}
-        run: |
-          echo "All good"
-      # ensure branch conaints only well-formed xml files 
+      # ensure branch contains only well-formed xml files 
```

Comment:
Is it no longer possible to have the Nothing Changed message?

Reply:
I moved the check higher up, in the file and paired it with a nicer log of changed files